cli/cli_utils.go

```diff
@@ -53,6 +53,72 @@ func randomizeClients(allClients clients.OrderedClients) (clients.Clients, error
 	return combinedClients, nil
 }
 
+func valClients(allClients clients.OrderedClients, flags *GenCmdFlags, services []string) (*clients.Clients, error) {
+	var executionClient, consensusClient, validatorClient *clients.Client
+	var err error
+
+	// execution client
+	if utils.Contains(services, execution) {
+		executionParts := strings.Split(flags.executionName, ":")
+		executionClient, err = clients.RandomChoice(allClients[execution])
+		if err != nil {
+			return nil, err
+		}
+		if flags.executionName != "" {
+			log.Warn(configs.CustomImagesWarning)
```

Comment:
You should only display this warning if there is a custom image (`strings.Join(executionParts[1:], ":")` not empty). Also, you know that in this case the custom image is for the execution client. The message is generic and mention the consensus and validator client.

Reply:
Done at 867bbeb021e05398e632c8dece2346444999decb